### PR TITLE
Fix rng tests

### PR DESCRIPTION
**Pull request by akharche**
- Fix test_uniform for #242 

- Fix assertion

### CODE REVIEW

**Comment by akharche** [comment]
@reazulhoque could you please look into this changes?

**Review by reazulhoque — CHANGES_REQUESTED** [review]

File: numba_dppy/tests/test_dpnp_functions.py
Comment (akharche) [reply numba_dppy/tests/test_dpnp_functions.py:1095]: Yes, test should pass.

In the provided example you compare boolean with float.
```
print(np.all(a) >= low)
print(np.all(a) < high)
```
We need to modify it in this way like I do in tests:

```
print(np.all(a >= low))
print(np.all(a < high))
```

Functions [np.all](https://numpy.org/doc/stable/reference/generated/numpy.all.html) and [np.ndarray.all](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.all.html#numpy.ndarray.all) are equal. Only the difference is that `np.all()` makes cast to np.array to call it for every array-like type. Calling np.ndarray.all may be slightly faster because it is built-in function for numpy arrays. I can modify tests if needed.

```
print((a >= low).all())
print((a < high).all())
```

File: numba_dppy/tests/test_dpnp_functions.py
Comment (akharche) [reply numba_dppy/tests/test_dpnp_functions.py:1090]: Yes, I found a mistake in check. Previously, I compared all values are >= 0. But standard_normal distribution returns values belong to R.

File: numba_dppy/tests/test_dpnp_functions.py
Comment (reazulhoque) [reply numba_dppy/tests/test_dpnp_functions.py:1095]: That makes perfect sense. Thank you for your input. In that case I think we do not need to make any changes to the checks.

**Review by reazulhoque — APPROVED** [review]

**Comment by reazulhoque** [comment]
@akharche Please merge with main. LGTM after the merge.

**Pull request merged**
